README.md

```diff
@@ -79,14 +82,14 @@ Your answer...
 You clone yet another Tunr repo. Put the following commands in the correct order necessary to make the app run. Delete the one command that will not be used.
 
 ```
-$ rake db:drop
-$ rake db:create
-$ rails new . -d postgresql
-$ bundle install
-$ git clone git@github.com:ga-wdi-exercises/moar-tunr.git
-$ rake db:migrate
-$ rails s
-$ rake db:seed
+3.$ rake db:drop
+4.$ rake db:create
+$ rails new . -d postgresql (you do not need this because you are not starting app from scratch)
+2.$ bundle install
+1.$ git clone git@github.com:ga-wdi-exercises/moar-tunr.git
+5.$ rake db:migrate
+7.$ rails s
+6.$ rake db:seed
```

Comment:
:white_check_mark: ...but next time, maybe just move them around on the page instead of giving them numbers! :stuck_out_tongue: